Hashlink.js

```diff
@@ -64,8 +64,14 @@ export class Hashlink {
     meta = {...meta, url: urls};
 
     // generate the encoded cryptographic hash
-    const outputData = await transforms.reduce(async (output, transform) =>
-      this.registeredTransforms[transform].encode(await output), data);
+    const outputData = await transforms.reduce(async (output, transform) => {
+      const transformer = this.registeredTransforms[transform];
+      if(transformer === undefined) {
+        throw new Error(`Unknown transformation ${transform}`)
```

Comment:
Maybe quote the name:
```suggestion
        throw new Error(`Unknown transformation "${transform}"`)
```

Reply:
Or maybe the style here is colon before unquoted user data?

Reply:
The above suggestion is good except for a missing period at the end.